docs/content/en/integrations/parsers/file/threagile.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Threagile threat modeling tool"
+toc_hide: true
+---
+Threagile JSON risks report format.
```

Comment:
Parser documentation is slowly being updated, to be a bit more useful with regards to what the report format should look like. [This is a great example](https://github.com/DefectDojo/django-DefectDojo/blob/bugfix/docs/content/en/integrations/parsers/file/aws_prowler_v3.md) of what we are looking for

Reply:
Will update it thank you!

Reply:
Updated document. PTaL. Thank you